### PR TITLE
Upgrade wallet libs

### LINKED ISSUE #585: After Disconnecting Wallet Connect Show Main Connection Modal
If you connect with WalletConnect, then disconnect, then click connect again, it will show you the wallet connect screen when it should show you:

Screen Shot 2023-08-07 at 5 33 57 PM

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- Upgrading the libraries improved the UX

Closes #585

### CODE REVIEW

File: umbra-js/tsconfig.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "composite": true,
     "declarationMap": true,
+    "skipLibCheck": true,
     "outDir": "build",
     "resolveJsonModule": true,
```
Comment (alexkeating) [inline umbra-js/tsconfig.json:8]: (arbitrary comment)

The logo is now messed up. Do we have a svg version of the logo.

Comment (alexkeating) [reply umbra-js/tsconfig.json:8]: It seems like the width is set to 56px. If we had an svg or a 56x56 image of the logo I think the issue would be resolved.

Comment (mds1) [reply umbra-js/tsconfig.json]: Which logo, the Umbra logo? If so I don't think we have one, I made the logo in powerpoint a few years ago and don't think I even have the original files anymore 😅 Might be able to import it into some image editing software and have it generate an SVG from it, but not sure

File: umbra-js/tsconfig.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
     "baseUrl": "./",
     "paths": {
       "@sinonjs/fake-timers": ["../node_modules/@types/sinonjs__fake-timers"]
-    }
+    },
+    "typeRoots": ["./node_modules/@types"]
```
Comment (alexkeating) [inline umbra-js/tsconfig.json:19]: Got the idea from here 

https://stackoverflow.com/a/76830746

File: frontend/src/store/wallet.ts [hunk 3]
```diff
@@ -132,9 +135,6 @@ export default function useWalletStore() {
             window.location.reload();
           });
           wallet.provider.on('chainChanged', () => {
-            if (wallet.label === 'WalletConnect' && !lastWallet.value) {
-              setLastWallet('WalletConnect');
-            }
```
Comment (mds1) [inline frontend/src/store/wallet.ts:137]: This was modified recently in https://github.com/ScopeLift/umbra-protocol/pull/559#discussion_r1253300467, why was it needed then but not now?

Comment (alexkeating) [reply frontend/src/store/wallet.ts:137]: When logging into wallet connect the page would reload before the last wallet was set which created a state where the user seemed to be logged out after the page reloaded. I tested and I wasn't able to create the issue the required this after upgrading the packages.

File: .github/workflows/ci.yaml [hunk 4]
```diff
@@ -31,13 +31,16 @@ jobs:
           submodules: recursive
```
Comment (apbendi) [inline .github/workflows/ci.yaml:31]: *Arbitrary comment location*

Testing with Rainbow on mobile and wallet connect on desktop, I get a "Connection Failed" and "Connection Rejected" message on each immediately after scanning the QR code. This does not happen in Prod, where I'm able to connect without issue.

Comment (alexkeating) [reply .github/workflows/ci.yaml:31]: It works with Metamask's wallet, and Rainbow works for me when I am running on my local computer, but I get the same error when I am using the deploy preview.

**Pull request merged**